### DISCUSSION TITLE
Homebrew formula for shell completion

### DISCUSSION THREAD

**Discussion by electriquo** [TomWright/dasel#284] [Ideas]
Dasel has a support for [shell completion](https://github.com/TomWright/dasel#completion) and has a [Homebrew formula for installation](https://github.com/TomWright/dasel#quickstart).

What do you think of adding another formula for installing the shell completion (such as [brew-cask-completion](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/brew-cask-completion.rb), [rake-completion](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/rake-completion.rb), [bundler-completion](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/bundler-completion.rb), [ruby-completion](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/ruby-completion.rb), [vagrant-completion](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/vagrant-completion.rb), [packer-completion](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/packer-completion.rb), [gem-completion](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/gem-completion.rb), [pip-completion](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/pip-completion.rb), ... just to name a few), or utilize the current formula for installing the shell completion?

**Comment by TomWright** [comment]
This would definitely make it easier to use, but it's a low priority for me right now since it's already possible to do.
I'll keep this open as a valuable idea and will work on it when I can.
